src/Application/Commands/UpdateCampaigns.php

```diff
@@ -61,7 +61,7 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 
             try {
                 $this->pull($campaign, $output);
-            } catch (NotFoundException $exception) {
+            } catch (NotFoundException|CampaignNotReady $exception) {
```

Comment:
should have been done earlier today when I introduced the CampaignNotReady exception class